models/junos/security_policy/junos_security_policies.yml

```diff
@@ -0,0 +1,514 @@
+            type: dict
+            suboptions:                  
+              alarm:
+                description:
+                  - The alarm for when a configured threshold is exceeded
+                type: dict
+                suboptions:
+                  per_minute_threshold:
+                    description:
+                      - threshold in kilobytes per minute
+                    type: int 
+                  per_second_threshold:
+                    description:
+                      - threshold in bytes per second
+                    type: int
+          
+          deny:
+            description: Block the service at the firewall. The device drops the packets.
+            type: bool
+            
+          reject:
+            description:
+              - Block the service at the firewall. The device drops the packet and sends a TCP reset (RST) segment to the source host for TCP traffic and an ICMP “destination unreachable, port unreachable” message (type 3, code 3) for UDP traffic.
+            type: dict
+            suboptions:
+              reject_enable:
+                description: flag to enable reject option
+                type: bool
+              reject_action:
+                description:
+                  - You can configure reject action with one of the following options for the dynamic_applications:
+                  - profile - You can chose to provide a notification to the clients or redirect client request to an informative Web page when a policy blocks HTTP or HTTPS traffic with a deny or reject action.
+                  - ssl_proxy - You can apply a redirect SSL proxy profile when a policy blocks HTTPS traffic with a reject action. When you apply am SSL proxy profile, SSL proxy decrypts the traffic and application identification functionality identifies the application. 
+                type: dict
+                suboptions:
+                  profile:
+                    description: You can chose to provide a notification to the clients or redirect client request to an informative Web page when a policy blocks HTTP or HTTPS traffic with a deny or reject action.
+                    type: str
+                  ssl_proxy:
+                    description: You can apply a redirect SSL proxy profile when a policy blocks HTTPS traffic with a reject action. When you apply am SSL proxy profile, SSL proxy decrypts the traffic and application identification functionality identifies the application.
+                    type: dict
+                    suboptions:
+                      profile_name:
+                        description: Name of ssl proxy profile
+                        type: str
+          
+          log:
```

Comment:
needs type

Reply:
ive used `choice` here. Do I need to include type for that as well?